.gitignore

```diff
@@ -1,2 +1,7 @@
 .DS_Store
 .tool-versions
+screenshots/
+./spec-sheets
+content-master/
+content-master
+screenshots/
```

Comment:
Review this list for duplicates and if they are actually required.

`content-master` ignores both a file and a directory named that, so the additional `content-master/` entry that would only ignore a directory is unnecessary and confusing.

`screenshots/` is in there twice, what are we actually trying to ignore?